src/extension/byok/node/openAIEndpoint.ts

```diff
@@ -81,18 +79,26 @@ export class OpenAIEndpoint extends ChatEndpoint {
 		);
 	}
 
-	override createRequestBody(options: ICreateEndpointBodyOptions): IEndpointBody {
-		options.ignoreStatefulMarker = false;
-		const body = super.createRequestBody(options);
-		if (this.useResponsesApi) {
-			body.store = true;
-			body.n = undefined;
-			body.stream_options = undefined;
-			if (!this._modelInfo.capabilities.supports.thinking) {
-				body.reasoning = undefined;
+	protected override preprocessOptions(options: ICreateEndpointBodyOptions): ICreateEndpointBodyOptions {
```

Comment:
This method is odd because the options came from this class, they came from `makeChatRequest`